engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
OK so assuming a JES backend with a preemptible limit of 3, does the following sound right?

If the JES backend runs attempt 3 on a preemptible VM that gets preempted, the backend should message the engine that the call has failed retriably.  The engine would launch the 4th attempt for the call, but now the JES backend would launch this attempt on a non-preemptible VM.